### ISSUE TITLE
Adding accessibilty features

### ISSUE THREAD

**Issue by Oxey405** [4ian/GDevelop#4525]
(I'm making a lot of issues this month because I got a few ideas for GDevelop)

## Description

Hey ! I think GDevelop is missing accessibility features.

## Solution suggested

Add new features for accessiblity :

- Colorblind white and dark css themes
- Audiodescription when events are hovered for 2+ seconds (needs a good open source text to speech)
- Dyslexic font implementation
- Font resize settings
- Nav. by using Tab key
- More things I didn't think about ?

## Alternatives considered

For the audiodescription, maybe Microsoft has a screen reader that can performs on GDevelop (needs testing)
Maybe find a nodejs module that could implement those features in electron apps ? (but how about web version)

**Comment by Silver-Streak** [comment]
Screen readers are generally going to be a system level option (and I can confirm that there is one native within Windows, as well as MacOS. I believe there are quite a few for Linux too), this is to ensure that a specific app's screenreader doesn't conflict/not adhere to the system level one's settings, which have been tailored to the specific user's visual/audio needs.

High contrast themes are probably a good callout, as is font resize settings.

Dyslexic font is an interesting one. [Recent studies have shown](https://www.edutopia.org/article/do-dyslexia-fonts-actually-work#:~:text=%E2%80%9CAll%20in%20all,%20the%20font%20Dyslexie,%20developed%20to,is%20printed%20in%20Arial%20or%20Times%20New%20Roman.%E2%80%9D) that they do not actually help with reading for those with dyslexia, but if it could be implemented I don't see why not have it as an option.

Definitely agree on font resizing.

The only other accessibility things I can think about are video object specific, which would be updating/getting a new video object that supports toggling embedded captions.
